modules/custom/ideal_blocks/ideal_blocks.module

```diff
@@ -158,7 +158,9 @@ function _ideal_blocks_idea_stats() {
 
     // Hardcode phase for now.
     $phase = t('Phase');
```

Comment:
We need to have a default Phase, probably through the field's configuration.
For now, change the initial value to "Ideation"
@israelLinnovate